### PR TITLE
Fix issues on press page

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: partials/archive-press/highlights.php [hunk 1]
```diff
@@ -79,9 +79,18 @@
...
-              <div class="font-size-h4"><?php _e('[:en]by[:es]por'); echo ' ' . $author; ?></div>
+              <div class="font-size-h4">
+                <?php 
+                  if (!empty($author)) {
+                    _e('[:en]by[:es]por'); 
+                    echo ' ' . $author;
+                  }
+                ?> 
+                <?php echo (!empty($date) && !empty($author) ? ' | ' : '' ); ?>
+                <?php _e(!empty($date) ? date('j F Y', $date) : '' ); ?>
```
Comment (varffs) [inline partials/archive-press/highlights.php]: this date isnt localized

**Comment by varffs** [comment]
I had quite a nice pattern going for localizing dates with the Gonzalo L project: https://github.com/interglobalvision/gonzalolebrija-com/blob/master/single-exposiciones.php#L5

However it requires Moment-php to be included so might be a faff at this point. Probably worth putting in the boiler tho

File: partials/archive-press/highlights.php (hunk 1 above)
Comment (mikeyrayvon) [inline partials/archive-press/highlights.php]: great catch 🏈

**Comment by varffs** [comment]
Yo @mikeyrayvon when you fix PRs can you remember to reassign plz so I can tell its ready in the list

**Pull request merged**